client/my-sites/backup/main.jsx

```diff
@@ -174,60 +175,58 @@ class BackupsPage extends Component {
 				<QuerySiteSettings siteId={ siteId } />
 				<QueryRewindCapabilities siteId={ siteId } />
 
-				<div className="backup__main-wrap">
-					<div className="backup__last-backup-status">
-						<BackupDatePicker
-							{ ...{
-								onDateChange: this.onDateChange,
-								selectedDate: this.getSelectedDate(),
-								siteId,
-								oldestDateAvailable,
-								today,
-								siteSlug,
-								dispatchRecordTracksEvent,
-							} }
-						/>
-
-						{ isLoadingBackups && <div className="backup__is-loading" /> }
-
-						{ ! isLoadingBackups && (
-							<>
-								<DailyBackupStatus
-									{ ...{
-										allowRestore,
-										siteUrl,
-										siteSlug,
-										backup: lastBackup,
-										lastDateAvailable,
-										selectedDate: this.getSelectedDate(),
-										timezone,
-										gmtOffset,
-										hasRealtimeBackups,
-										onDateChange: this.onDateChange,
-										deltas,
-										// metaDiff, todo: commented because the non-english account issue
-										doesRewindNeedCredentials,
-										dispatchRecordTracksEvent,
-									} }
-								/>
-							</>
+				{ isLoadingBackups && <BackupPlaceholder /> }
```

Comment:
This confused me for a while because I couldn't understand why we needed to put a placeholder here when we already have that configured in the controller. So, I removed it to see what was it purpose and I realized we clearly need both :)